docs/runbooks/KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert.md

```diff
@@ -21,4 +21,4 @@ spec: {}
 EOF
 ```
 
-In case of uninstallation, if HCO is still installed and running for more than an hour after removing the `HyperConverged` custom resource, completed the process by uninstall HCO. If for some reason, HCO is still running (uninstallation is stuck), this issue should be solved in order to cancel the alert.
+In case of uninstallation, if HCO is still installed and running for more than an hour after removing the `HyperConverged` custom resource, complete the process by uninstall HCO. If for some reason, HCO is still running (uninstallation is stuck), this issue should be solved in order to cancel the alert.
```

Comment:
@orenc1 If you are already fixing it I would change:
complete the process by uninstall HCO => complete the process by **uninstalling** HCO 
or complete the process of HCO uninstallation

Reply:
fixed.